pkg/ccl/backupccl/backup_job.go

```diff
@@ -1230,8 +1248,16 @@ func (b *backupResumer) OnFailOrCancel(
 }
 
 // CollectProfile is a part of the Resumer interface.
-func (b *backupResumer) CollectProfile(_ context.Context, _ interface{}) error {
-	return nil
+func (b *backupResumer) CollectProfile(ctx context.Context, execCtx interface{}) error {
+	p := execCtx.(sql.JobExecContext)
+
+	// TODO(adityamaru): We should move the logic that decorates the DistSQL
+	// diagram with per processor progress to this method.
+
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	return bulkutil.FlushTracingAggregatorStats(ctx, b.job.ID(),
```

Comment:
This is potentially a going to hold the `mu` for some time. Could consider copying the map -- both levels -- to release the mu before passing the copy to flush, since the mu blocks draining the progress ch which can backpressure the flow can't it?

Reply:
Oh interesting so if we don't flush tracingAggCh fast enough, we could backup the processors from sending progress metas back to us. I didn't think of that. I'm cool with copying the map and passing that in to flush.